### ISSUE TITLE
Timeline view for media  

### ISSUE THREAD

**Issue by timaniac00** [UnevenSoftware/LeafPic#270]
Please, allow to view all images sorted by date like into Pictures, Google Photo and HTC Gallery apps

**Comment by sriramr98** [comment]
I would like to take on the feature request. Whom should I contact to get started?

**Comment by dnldsht** [comment]
That feature was partially implemented by #423 you may improve it adding
some sort of fast scrolling or group images by date

**Comment by sriramr98** [comment]
@dnldsht  Yeah I have some ideas. However I need help with getting started as I've never contributed to an open source project anymore.

**Comment by calvinnor** [comment]
@dnldsht I suppose we can close this now? (After 2 long years)

Or do you want to wait to stabilise it and remove the Debug flag, and then we'll close?
